latex/Research_Strategy.tex

```diff
@@ -227,27 +233,29 @@
 %c. Mention uniqueness of SAMPL -- there are other predictive challenges (D3R, pKa coop, CASP) but none focused on driving quantitatively predictive protein-ligand modeling
 This work is also innovative because of the uniqueness of SAMPL.
 While there are other predictive challenges in the area of biomolecular modeling, such as D3R [ref], the pKa cooperative [ref], and CASP [ref],
+{\color{red}[JDC: Is CAPRI still a thing? Are there other docking or ligand binding prediction challenges?]}
 %insert references
 none are specifically focused on driving quantitatively predictive protein-ligand modeling.
-SAMPL is unique because -- at least with the deliberate design plan we propose here -- it will be specifically tailored in order to drive improvements in modeling for biomolecular design.
+The SAMPL expansion we propose here is unique because it will be \emph{specifically designed} to drive improvements in modeling accuracy for biomolecular design, rather than simply plan an evaluative role on targets of pharmaceutical interest.
```

Comment:
Grammar issues, I think. Maybe, "The SAMPL expansion we propose here is unique because it will be \emph{specifically designed} to drive improvements in modeling accuracy for biomolecular design, rather than simply serving [maybe you meant "playing an evaluative role"?] to evaluate accuracy on targets of pharmaceutical interest."